### ISSUE TITLE
[MAS2.4.7] Focus indicator is not visible on "Upload File" and "Send" buttons.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1306]
**Actual Result:**
Keyboard focus indicator is not visible on "Upload card" and "Send" buttons.

**Note:**
Same issue repro at -
1. Url -> Type "accessibility" in text box -> Press Enter -> Navigate to "New message" button on the page by using Tab key - Focus indicator is not visible on “New Messages” button. Please refer the attachment **"MAS2.4.7_New message button."**

**Expected Result:**
Keyboard focus indicator should be visible on "Upload card" and "Send" buttons.

**Repro Steps:**
1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle](https://microsoft.github.io/BotFramework-WebChat/full-bundle) in Edge browser.
2. Notice 'Upload File' and 'Send' button icons present in bottom of screen.
3. Navigate to Upload File' and 'Send' buttons by using Tab key.
4. Observe that focus indicator is not visible on "Upload File" and "Send" buttons.

**User Impact:**
If focus indicator is not visible on any control then the user who is visually impaired will not able to understand the landing of focus.

**MAS Reference:**
[MAS 2.4.7 - Focus Visible (05A)](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c)

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17134.345
**OS Version:** 1803
**Browser:** Edge

**Attachment:**
1. [MAS2.4.7_Focus indicator _Upload file and send button.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2547055/MAS2.4.7_Focus.indicator._Upload.file.and.send.button.zip)
2. [MAS2.4.7_New message button.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2547279/MAS2.4.7_New.message.button.zip)

**Comment by priysn** [comment]
As part of internal bug review, closing this bug.

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG2.4.7
